### PR TITLE
Ticket/3790 nci hero analytics

### PR DESCRIPTION
**Pull request by dlescarbeau**

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/nci-hero/nci-hero.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+};
+
+/**
+ * Click handler for the language toggle click.
+ * @param promoBlockRow the promo-block element.
+ */
+const nciHeroLinkClickHandler =
+	(nciHero: HTMLElement, totalLinks: number) => (evt: Event) => {
+		const target = evt.currentTarget as HTMLElement;
+		evt.preventDefault();
```
Comment (olitharp-nci) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/nci-hero/nci-hero.ts:76]:
```suggestion
```

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/nci-hero/nci-hero.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	} else if (anchorElementParent) {
+		return `2-${ctaStripElementArray.indexOf(anchorElementParent) + 1}`;
+	} else {
+		return '_ERROR_';
+	}
+};
+
+/**
+ * Click handler for the language toggle click.
+ * @param promoBlockRow the promo-block element.
```
Comment (olitharp-nci) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/nci-hero/nci-hero.ts:71]: Update jsdoc for hero

**Review by olitharp-nci — CHANGES_REQUESTED** [review]
Update your title and description also

**Review by olitharp-nci — APPROVED** [review]

**Comment by bennettcc** [comment]
This was approved by analytics team on 6/5 - adding product checkmark on their behalf

**Review by bennettcc — APPROVED** [review]

**Pull request merged**